addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -1,15 +1,33 @@
 /* @odoo-module */
 
-import { Component } from "@odoo/owl";
+import { Component, useState } from "@odoo/owl";
 import { useRtc } from "@mail/rtc/rtc_hook";
 import { isMobileOS } from "@web/core/browser/feature_detection";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { useService } from "@web/core/utils/hooks";
+import { _t } from "@web/core/l10n/translation";
 
 export class CallActionList extends Component {
+    static components = { Dropdown, DropdownItem };
     static props = ["thread", "fullscreen", "compact?"];
     static template = "mail.CallActionList";
 
     setup() {
         this.rtc = useRtc();
+        this.userSettings = useState(useService("mail.user_settings"));
+    }
+
+    get moreOptions() {
+        const options = [];
+        if (this.env.debug && this.userSettings.logRtc) {
+            options.push({
+                icon: "fa fa-lg fa-file-text-o",
+                label: _t("Download logs"),
```

Comment:
How about "Download RTC logs"? The term "Logs" is perhaps a bit misleading.

Reply:
Just logs is good, I really don't like using `RTC` to the user (even in our code, "call" is more appropriate)